app/views/barbeque/monitors/index.html.haml

```diff
@@ -0,0 +1,51 @@
+.row
+  .col-sm-12
+    .box.box-primary
+      .box-header
+        %h3.box-title.with_padding Recently processed jobs (hourly)
+      .box-body
+        #recently-processed-jobs-chart{data: { jobs: @recently_processed_jobs.to_json }}
+        %table.table
+          %thead
+            %tr
+              %th Hour
+              %th Application
+              %th Job
+              %th Count
+          %tbody
+            - @recently_processed_jobs.keys.sort { |x, y| y <=> x }.each do |date_hour|
+              - jobs = @recently_processed_jobs[date_hour]
+              - jobs.keys.sort.each do |job_id|
+                - job = jobs[job_id]
+                %tr
+                  %td= date_hour
+                  %td= link_to(job[:app_name], app_path(job[:app_id]))
+                  %td= link_to(job[:job_name], job_definition_path(job[:job_id]))
+                  %td= job[:count]
+
+:coffee
```

Comment:
Fixed in https://github.com/cookpad/barbeque/pull/19/commits/1b0e2cf3934f28733a273169864d9a52e8696ea8